packages/grid/src/Grid.tsx

```diff
@@ -702,6 +713,9 @@ export const Grid = function Grid<T>(props: GridProps<T>) {
       if (part === "header" && !headerIsFocusable) {
         return;
       }
+      if (editMode) {
```

Comment:
What if click on header or other intractable elements outside of the grid?

Reply:
Yeah, I was thinking of setting up a global click listener but then realised it will have conflicts with existing listeners. Also, Im pretty sure calling `.stopPropagation()` on a react event will not stop it from bubbling to the actual dom listeners? Maybe we handle this in next iteration.

Reply:
But to answer your question, nothing 😅

Reply:
Actually, I think I found a solution. Stay tuned.

Reply:
@origami-z Ok, I have now updated the PR with a solution for this. Check out the [listen to click on the page and on the header](https://github.com/jpmorganchase/salt-ds/pull/1462/commits/b6e6bd7c42965576f237e5ab45acfa00597c733c) commit.